src/test/web-platform-tests/run-all.js

```diff
@@ -105,8 +114,9 @@ const skip = [
     "transaction-abort-request-error.js",
 ];
 
-const filenames = fs.readdirSync(testFolder);
-for (const filename of filenames) {
+const filenames = glob.sync(testFolder + "/**/*.js");
```

Comment:
Why switch from readdirSync to glob? All else equal, it'd be better not to have another dependency.

If all else is not equal and there is a good reason for glob, add it to package.json in devDependencies. This is currently just working by chance, since glob is a dependency of a dependency.

Reply:
glob lets it recurse into subdirectories. Without, it misses the `key-generators` tests.

Good point on the dependency - I keep thinking it's built into node, but of course it isn't. I also realised that this way of constructing the pattern is unsafe, so I'll fix that.